### DISCUSSION TITLE
Webinterface updaten ?

### DISCUSSION THREAD

**Discussion by bmause** [jomjol/AI-on-the-edge-device#2593] [Q&A] [answered]
Haloo zusammen, 
ich habe über das OTA Update heute auf die Version 15.3 das update gemacht.
Die Firmware.bin geht auch recht einfach, aber wie lade ich die Config & html hoch?
Muss ich da wirklich jede Datei einzeln auswählen?

Nach dem Reboot kommt jetzt das das Webinterface noch auf 15.2.4 steht.

Wie date ich das den ab ?

<img width="573" alt="Bildschirmfoto 2023-08-31 um 12 55 47" src="https://github.com/jomjol/AI-on-the-edge-device/assets/105861086/bdc1ffaa-c839-4a52-88a4-b3bbfc3c042b">

**Comment by Slider0007** [comment, marked as answer]
Hallo,
du musst nicht die firmware.bin extrahieren und auswählen, sondern direkt das update.zip File beim OTA auswählen, dann wird alles auf einmal upgedatet.
-> OTA mit zip file wiederholen

**Reply by bmause** [reply]
danke, da hatte ich den Fehler gemacht. Der Mac entpackt gleich nach den Download :-)
